src/core/jsonschema/frame.cc

```diff
@@ -90,9 +91,10 @@ auto find_anchors(const sourcemeta::core::JSON &schema,
       assert(schema.at("id").is_string());
       const sourcemeta::core::URI identifier(schema.at("id").to_string());
       if (identifier.is_fragment_only()) {
```

Comment:
```suggestion
      if (identifier.is_fragment_only() && identifier.fragment().has_value()) {
```

Same here. It is a bit redundant, but this could make ClangTidy happy